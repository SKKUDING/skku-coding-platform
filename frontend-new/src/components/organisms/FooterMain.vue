<template>
  <div class="flex flex-col items-center gap-y-3 text-stone-400 my-6">
    <div class="flex gap-x-6">
      <AtomsFooterIconMail />
      <AtomsFooterIconGitHub />
      <AtomsFooterIconKakao />
      <AtomsFooterIconLink />
    </div>
    Â© 2011-2022 SKKU NPC
  </div>
</template>
