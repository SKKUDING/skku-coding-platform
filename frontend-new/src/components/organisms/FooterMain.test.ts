import { describe, expect, test } from 'vitest'
import { render } from '@testing-library/vue'
import FooterMain from './FooterMain.vue'

describe('Atom - FooterMain', () => {
  test('should render', () => {
    const { html } = render(FooterMain)
    expect(html()).toMatchSnapshot()
  })
})
