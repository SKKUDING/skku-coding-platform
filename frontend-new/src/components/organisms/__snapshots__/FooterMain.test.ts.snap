// Vitest Snapshot v1

exports[`Atom - FooterMain > should render 1`] = `
"<div class=\\"flex flex-col items-center gap-y-3 text-stone-400 my-6\\">
  <div class=\\"flex gap-x-6\\"><a href=\\"mailto:npc.skku@g.skku.edu\\"><svg width=\\"1.2em\\" height=\\"1.2em\\" preserveAspectRatio=\\"xMidYMid meet\\" viewBox=\\"0 0 512 512\\">
        <path d=\\"M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7c22.4 17.4 52.1 39.5 154.1 113.6c21.1 15.4 56.7 47.8 92.2 47.6c35.7.3 72-32.8 92.3-47.6c102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4c132.7-96.3 142.8-104.7 173.4-128.7c5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9c30.6 23.9 40.7 32.4 173.4 128.7c16.8 12.2 50.2 41.8 73.4 41.4z\\" fill=\\"currentColor\\"></path>
      </svg></a><a href=\\"https://github.com/skku-npc/skku-coding-platform\\" target=\\"_blank\\"><svg width=\\"1.2em\\" height=\\"1.2em\\" preserveAspectRatio=\\"xMidYMid meet\\" viewBox=\\"0 0 1536 1504\\">
        <path d=\\"M768 0q209 0 385.5 103T1433 382.5T1536 768q0 251-146.5 451.5T1011 1497q-27 5-40-7t-13-30q0-3 .5-76.5t.5-134.5q0-97-52-142q57-6 102.5-18t94-39t81-66.5t53-105T1258 728q0-119-79-206q37-91-8-204q-28-9-81 11t-92 44l-38 24q-93-26-192-26t-192 26q-16-11-42.5-27T450 331.5T365 318q-45 113-8 204q-79 87-79 206q0 85 20.5 150T351 983t80.5 67t94 39t102.5 18q-39 36-49 103q-21 10-45 15t-57 5t-65.5-21.5T356 1146q-19-32-48.5-52t-49.5-24l-20-3q-21 0-29 4.5t-5 11.5t9 14t13 12l7 5q22 10 43.5 38t31.5 51l10 23q13 38 44 61.5t67 30t69.5 7t55.5-3.5l23-4q0 38 .5 88.5t.5 54.5q0 18-13 30t-40 7q-232-77-378.5-277.5T0 768q0-209 103-385.5T382.5 103T768 0zM291 1103q3-7-7-12q-10-3-13 2q-3 7 7 12q9 6 13-2zm31 34q7-5-2-16q-10-9-16-3q-7 5 2 16q10 10 16 3zm30 45q9-7 0-19q-8-13-17-6q-9 5 0 18t17 7zm42 42q8-8-4-19q-12-12-20-3q-9 8 4 19q12 12 20 3zm57 25q3-11-13-16q-15-4-19 7t13 15q15 6 19-6zm63 5q0-13-17-11q-16 0-16 11q0 13 17 11q16 0 16-11zm58-10q-2-11-18-9q-16 3-14 15t18 8t14-14z\\" fill=\\"currentColor\\"></path>
      </svg></a><a href=\\"http://pf.kakao.com/_UKraK/chat\\" target=\\"_blank\\"><svg width=\\"1.2em\\" height=\\"1.2em\\" preserveAspectRatio=\\"xMidYMid meet\\" viewBox=\\"0 0 24 24\\">
        <path d=\\"M12 3c5.799 0 10.5 3.664 10.5 8.185c0 4.52-4.701 8.184-10.5 8.184a13.5 13.5 0 0 1-1.727-.11l-4.408 2.883c-.501.265-.678.236-.472-.413l.892-3.678c-2.88-1.46-4.785-3.99-4.785-6.866C1.5 6.665 6.201 3 12 3zm5.907 8.06l1.47-1.424a.472.472 0 0 0-.656-.678l-1.928 1.866V9.282a.472.472 0 0 0-.944 0v2.557a.471.471 0 0 0 0 .222V13.5a.472.472 0 0 0 .944 0v-1.363l.427-.413l1.428 2.033a.472.472 0 1 0 .773-.543l-1.514-2.155zm-2.958 1.924h-1.46V9.297a.472.472 0 0 0-.943 0v4.159c0 .26.21.472.471.472h1.932a.472.472 0 1 0 0-.944zm-5.857-1.092l.696-1.707l.638 1.707H9.092zm2.523.488l.002-.016a.469.469 0 0 0-.127-.32l-1.046-2.8a.69.69 0 0 0-.627-.474a.696.696 0 0 0-.653.447l-1.661 4.075a.472.472 0 0 0 .874.357l.33-.813h2.07l.299.8a.472.472 0 1 0 .884-.33l-.345-.926zM8.293 9.302a.472.472 0 0 0-.471-.472H4.577a.472.472 0 1 0 0 .944h1.16v3.736a.472.472 0 0 0 .944 0V9.774h1.14a.472.472 0 0 0 .472-.472z\\" fill=\\"currentColor\\"></path>
      </svg></a><a href=\\"https://npc.skku.edu\\" target=\\"_blank\\"><svg width=\\"1.2em\\" height=\\"1.2em\\" preserveAspectRatio=\\"xMidYMid meet\\" viewBox=\\"0 0 512 512\\">
        <path d=\\"M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59c-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0c-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606c.648 17.722 3.826 35.527 9.69 52.721c1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96c28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83c-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37c-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569c-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51c27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612c5.864 17.194 9.042 34.999 9.69 52.721c.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z\\" fill=\\"currentColor\\"></path>
      </svg></a></div> Â© 2011-2022 SKKU NPC
</div>"
`;
